css/properties/text-decoration-inset.json

```
@@ -0,0 +1,76 @@
+{
+  "css": {
+    "properties": {
+      "text-decoration-inset": {
+        "__compat": {
+          "spec_url": "https://drafts.csswg.org/css-text-decor-4/#text-decoration-inset-property",
+          "tags": [
+            "web-features:text-decoration"
+          ],
+          "support": {
+            "chrome": {
+              "version_added": false
+            },
+            "chrome_android": "mirror",
+            "edge": "mirror",
+            "firefox": {
+              "version_added": "146",
+              "impl_url": "https://bugzil.la/1993043"
```

We can remove the bug now:

```suggestion
              "version_added": "146"
```